ush/machine/cheyenne.yaml

```diff
@@ -16,6 +16,7 @@ platform:
   RUN_CMD_POST: mpirun -np $nprocs
   RUN_CMD_SERIAL: time
   RUN_CMD_UTILS: mpirun -np $nprocs
```

Comment:
@padhrigmccarthy, I think the `RUN_CMD_AQM` command line is necessary. How did you run the `nexus_emission` task without this command on Chyeynne?

Reply:
@chan-hoo I didn't include my  complete ush/machine/cheyenne.yaml here, I simply injected the changes for this issue into the (admittedly incomplete) cheyenne configuration here. I plan to contribute my entire cheyenne configuration as a separate PR soon.

Reply:
Got it!